components/kyma-environment-broker/internal/runtime/handler.go

```diff
@@ -138,7 +138,7 @@ func (h *Handler) determineStatusModifiedAt(dto *pkg.RuntimeDTO) error {
 	// Determine runtime modifiedAt timestamp based on the last operation of the runtime
 	last, err := h.operationsDb.GetLastOperation(dto.InstanceID)
 	if err != nil && !dberr.IsNotFound(err) {
-		return errors.Wrapf(err, "while fetching last operation for instance %s", dto.InstanceID)
+		return fmt.Errorf("while fetching last operation for instance %s", dto.InstanceID)
```

Comment:
need wrap this, otherwise, we can't get err_category and err_reason from `func ReasonForError(err error) LastError`.
The error category and error reason from LastError will be extracted and displayed in alert `ProvisionFailure` and `DeprovisionFailure`. for example https://sap-btp.slack.com/archives/CSR4HDWEL/p1666599864960759